clj/src/cljd/test.cljd

```diff
@@ -395,11 +396,19 @@
   (let [v (volatile! nil)]
     [#(vreset! v (f)) #(f @v)]))
 
-(defn register-fixtures [fixtures-callbacks]
+(defn register-once-fixtures [fixtures-callbacks]
```

Comment:
For background: in an earlier version for cljd.test, assertions were mapped to dart tests and tests to dart groups.
Since setUpAll and tearDownAll are scoped by groups they worked for `:each` fixtures too. We never realized we broke this when aligning cljd tests with dart tests.